lib/Optimizer/CodeGen/LowerToBaseProfileQIR.cpp

```diff
@@ -199,21 +199,23 @@ struct AddFuncAttribute : public OpRewritePattern<LLVM::LLVMFuncOp> {
     rewriter.startRootUpdate(op);
     const auto &info = iter->second;
     // QIR functions need certain attributes, add them here.
+    // TODO: Update schema_id with valid value
     auto arrAttr = rewriter.getArrayAttr(ArrayRef<Attribute>{
-        rewriter.getStringAttr("EntryPoint"),
+        rewriter.getStringAttr("entry_point"),
+        rewriter.getStrArrayAttr({"qir_profiles", "base_profile"}),
+        rewriter.getStrArrayAttr({"output_labeling_schema", "schema_id"}),
         rewriter.getStrArrayAttr(
+            // TODO: change to required_num_qubits once providers support it
```

Comment:
For todo's and fixme's in the code, please file a GitHub issue and add a link to it in the comment to make sure they don't get forgotten.

Reply:
Addressed in 96567e50. (They already had existing issues, so I just referenced them.)